src/components/Chart/Maps.js

```diff
@@ -189,6 +226,70 @@ export class GenericMap extends Component {
       })
     })
   }
+  renderPointTooltip() {
+    const {
+      hoverPoint,
+      layout: {
+        projectPoint,
+        numberFormat
+      }
+    } = this.state
+    const {
+      width,
+      pointLabel,
+      pointAttributes,
+      unit,
+    } = this.props
+    
+    if (!hoverPoint) {
+      return null
+    }
+
+    const [ x, y ] = projectPoint([hoverPoint.datum.lon, hoverPoint.datum.lat])
+    
+    const value = isNaN(hoverPoint.datum.value) 
+      ? String(hoverPoint.datum.value).trim()
+      : numberFormat(hoverPoint.datum.value)
+
+    const body = pointAttributes.map(t => {
+      const val = hoverPoint.datum[t]
+      if (val) {
+        return (<>{subsup(val)}<br/></>)
+      } else {
+        return null
+      }
+    })
+
+    const hasValue = hoverPoint.datum.value !== undefined
+    const showTooltip = hasValue || body.length > 0 || hoverPoint.datum[pointLabel]
+
+    if (showTooltip) {
+      return (
+        <ContextBox
+          orientation="top"
+          x={x}
+          y={y}
+          contextWidth={width}
+        >
+          <ContextBoxValue label={hoverPoint.datum[pointLabel]}>
+            <div {...styles.tooltip}>
```

Comment:
`<ContextBox>` already sets fontSize 14 and `WebkitUserSelect` is not needed here right? IMO rm.